rspec_log.gemspec

```diff
@@ -16,10 +16,12 @@ Gem::Specification.new do |s|
   s.authors     = ['Andrea Capri']
   s.email       = ['andrea.a.capri@gmail.com']
 
-  s.add_development_dependency 'rake'
-  s.add_development_dependency 'rspec', '~> 3.0'
-
+  s.add_runtime_dependency 'rake'
```

Comment:
Since this module doesnt actually use most of these dependencies at runtime it would be better to keep them as development dependencies.